### ISSUE TITLE
is this project still actively maintained?

### ISSUE THREAD

**Issue by m1cha3lf** [Snapchat/KeyDB#839]
Hi,

i'm wondering if this project is still actively maintained or not. I see some really critical bug reports but no activity from existing maintainers. 

As it's nature a database needs a reliable environment and consistency in its development.

I'm a happy user of KeyDB but concerned about the existing issues and uncertainty.

If it was just a wrong impression feel free to delete this issue.

michael

**Comment by mbfdias** [comment]
there was a recent issue (duplicate of this and closed by @JohnSully ) #788 

don't know if it answers your question :)

**Comment by m1cha3lf** [comment]
Hi,

thanks! I saw this issue as well, but created a new one because since then there was no activity in this project at all. No reaction to critical bug reports etc. I think the situation is a bit different now. 

Given the time of  almost 2 weeks without a response from a maintainer confirms what was expected i think...

**Comment by kanpov** [comment]
Yeah, I'm probably just going to use Valkey or even something more "cutting-edge" like [Garnet](https://github.com/microsoft/garnet) since KeyDB seems to be mostly abandoned.

**Comment by ilyavaiser** [comment]
This project is dead now and have some really critical bugs, don't use it in production

**Comment by m1cha3lf** [comment]
does anyone know why it's dead? I mean it's still Snap behind who might have an interest to keep it running!?

**Comment by osevan** [comment]
Im still using in production systems.

**Comment by williamstein** [comment]
> This project is dead now and have some really critical bugs, don't use it in production

See comments from the lead developer last week to understand their development and maintenance priorities: https://github.com/Snapchat/KeyDB/issues/798#issuecomment-2214123117

**Comment by binaryfire** [comment]
I just looked at the activity of the 2 main contributors. It appears one is now contributing to Valkey and the other hasn't been online much since June. So yeah, it seems development has definitely stalled. It's a real shame - the `EXPIREMEMBERS` command is a gamechanger for tagged caching. But I'd advise against using KeyDB in production going forward.

**Comment by n-rodriguez** [comment]
Hi there! Any news? It seems this project is definitely dead with the departure of @JohnSully  (see: https://github.com/Snapchat/KeyDB/issues/895)

**Comment by mbfdias** [comment]
@n-rodriguez @binaryfire @williamstein @osevan @m1cha3lf @ilyavaiser @kanpov 

There is an ongoing project to add support for multi master to valkey if you want to test it, the code has already been through a round of testing and it's working quite well. Some more testers would be great

you can download from: https://github.com/bet0x/valkey, please do read https://github.com/bet0x/valkey/blob/unstable/valkey-multimaster-setup.md for instructions on setup

**Comment by Lazerbeak12345** [comment]
#923
